package.json

```diff
@@ -60,7 +60,7 @@
     "npm": "^2.14.0"
   },
   "scripts": {
-    "build": "gulp",
+    "build": "git clone -b fix\/tracker-ebola git://github.com/frictionlessdata/jsontableschema-js.git node_modules/jsontableschema && cd node_modules/jsontableschema && npm install && npm run build && gulp",
```

Comment:
Oh, understood it now! You're building the actual jsontableschema. You should be able to add a GitHub repository as a dependency in `package.json`, so hopefully there would be no need to build `jsontableschema` here. Also, I'm not sure if this is the intended behaviour, but `gulp` is running inside `node_modules/jsontableschema`.